docs/api/products.rst

```diff
@@ -0,0 +1,27 @@
+========
+Products
+========
+
+This allows list, retrieve, create, update, and delete of Products.
+
+Retrieve
+--------
+
+.. sourcecode:: http
+
+   GET /api/products/ HTTP/1.1
+
+.. sourcecode:: http
+
+   HTTP/1.1 200 OK
+
+   [
+      {
+        "id": "3Lw7OwK",
+        "name": "Test Product",
+        "repo_name": "test-product",
+        "version_number": "v0.1.0",
+        "description": "<p>This is safely rendered markdown</p>",
+        "is_managed": false,
+      }
+   ]
```

Comment:
@wlonk I think we might as well paginate these right from the beginning. Similar to MD, but no need for the `first_page` or category meshugas.

Reply:
Is pagination trivial? If not, leave it for later.

Reply:
Because we're copying over some patterns from MD, I'd say on the front end it's trivial (arguably easier than ripping it out). I'll let @wlonk speak to the back end.